index.html

```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
+                <span class="project-view-union">See this project</span>
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+    <div class="projects-wrapper">
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21 (4).png" alt="image">
+        </div>
+        <div class="project-lg">
+          <div class="project-lg-1">
+            <h3>Project name goes here</h3> 
+            
+            <button type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
+            
+            <button type="submit" id="project-view">
+                <span class="project-view-union">See this project</span>
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+    <div class="projects-wrapper">
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21 (5).png" alt="image">
+        </div>
+        <div class="project-lg">
+          <div class="project-lg-1">
+            <h3>Project name goes here</h3> 
+            
+            <button type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
+            
+            <button type="submit" id="project-view">
+                <span class="project-view-union">See this project</span>
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+  </section>
+  <section class="about">
+    <div class="about-me">
+        <h1>About</h1>
+        <P>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</P>
+
+        <button type="submit" id="about-view">
+            <span class="project-view-btn">Get my CV</span>
+            
+            <span><img id="cv-btn" src="images/ic_download.png" alt="union"></span>
+        </button>
+    </div>
+    <section class="langs-section">
+        <div class="langs">
+           <div class="content-list">
+            <h3>Languages</h3>
+            <ul>
+                <li>JavaScript (ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
```

Comment:
- [ ] Please ensure that the technology categories ("Languages", "Frameworks" and "Skills") are `<li>` items enclosed within a `<ul>`. This is the right way to markup a list of things. 👍